### ISSUE TITLE
Metal Caster, Iron Gear, Catch 22

### ISSUE THREAD

**Issue by Kenkron** [Forecaster/RailcraftPatronServerIssues#24]
In 1.15

As I understand it, in order to make a *Metal Caster*, the recipe should require two *Brass Gears*. Instead, the recipe requires two *Iron Gears*.

![screenshot_20170525_082430](https://cloud.githubusercontent.com/assets/1354242/26451695/1f29aa58-412b-11e7-8066-01a0f9c93d04.png)

Attempting to use *Brass Gears* doesn't work.

![screenshot_20170525_090900](https://cloud.githubusercontent.com/assets/1354242/26451801/86ca2e76-412b-11e7-9110-5b4fe1831be3.png)

This is a problem because the only way to get *Iron Gears* is to make them in a *Metal Caster*. But if the *Metal Caster* requires *Iron Gears* to make, so you can't make a metal caster unless you already have a metal caster. Catch 22.

I encountered this two days ago, and Ash gave me some iron gears to get me started. Then, yesterday, the recipe switched to brass gears, so I crafted two of those. Now, today (25 May 2017), it's back to iron gears.

**Comment by Kenkron** [comment]
UPDATE:

While the iron gear recipe *LOOKS* like it works, it actually doesn't.  I can't pull it out of the crafting table.

**Comment by CovertJaguar** [comment]
Could it be that you didn't put the craft tweaker scripts folder in the root directory? And your client is using the wrong recipes.

I made one the other day (on 1.15) and it used Brass Gears just fine.

**Comment by Kenkron** [comment]
I did a clean download again, and it worked a bit, then didn't work a bit. I have no real idea why it's acting up, but I wonder if there is a race condition during loading.  Like, JEI has to look at item recipes, and CraftTweaker has to change recipes, and maybe they're not doing it in the right order all the time.

**Comment by liach** [comment]
As far as I know, recipes are determined by the server, without client modifications necessary

**Comment by CovertJaguar** [comment]
@liach this is true, but JEI is clientside, and it won't actually display the output if the client has different recipes, even if it will craft it.
